src/nbt/tag.rs

```diff
@@ -74,29 +84,13 @@ impl NbtTag {
         bytes.freeze()
     }
 
-    /// Serializes as single NBT tag without name.
-    pub fn serialize_tag(&self) -> Bytes {
-        let mut bytes = BytesMut::new();
-        bytes.put_u8(self.id());
-        bytes.put(self.serialize_raw());
-        bytes.freeze()
-    }
-
-    /// Serializes the NBT tag into bytes with a name and id.
-    pub fn serialize_named(&self, name: &str) -> Bytes {
-        let mut bytes = BytesMut::new();
-        bytes.put_u8(self.id());
-        bytes.put(NbtTag::String(name.to_string()).serialize_raw());
-        bytes.put(self.serialize_raw());
-        bytes.freeze()
-    }
-
     pub fn deserialize(bytes: &mut Bytes) -> Result<NbtTag, Error> {
         let tag_id = bytes.get_u8();
-        Self::deserialize_raw(bytes, tag_id)
+        Self::deserialize_data(bytes, tag_id)
     }
 
-    pub fn deserialize_raw(bytes: &mut Bytes, tag_id: u8) -> Result<NbtTag, Error> {
+    /// Deserializes tag data from bytes.
+    pub fn deserialize_data(bytes: &mut Bytes, tag_id: u8) -> Result<NbtTag, Error> {
```

Comment:
Ladies and gentelmen again why there is feature in rust like function name's when we have comments